src/shared_modules/utils/timeHelper.h

```diff
@@ -19,6 +19,22 @@
 #include <sstream>
 #include <string>
 
+#ifdef WIN32
+
+static struct tm* gmtime_r(const time_t* timep, struct tm* result)
+{
+    errno = gmtime_s(result, timep);
```

Comment:
```suggestion
    errno_t err = gmtime_s(result, timep);
    return err == 0 ? result : nullptr;
```

Reply:
In this case, we intend for our `gmtime_r` function to behave like the original POSIX function: [manual](https://man7.org/linux/man-pages/man3/gmtime.3p.html):

> Upon successful completion, the gmtime() function shall return a
 pointer to a `struct tm`.  If an error is detected, gmtime() shall
 return a null pointer and set [errno](https://man7.org/linux/man-pages/man3/errno.3.html) to indicate the error.

That's why the function is writing the global variable `errno`.